src/components/Insights/ActivityFeed.js

```diff
@@ -243,16 +243,10 @@ export const ActivityFeed = React.createClass({
   fetchSentences(requestPayload, callback){
       let {categoryValue, timespanType, searchValue, limit, offset, edges, siteKey,
            categoryType, filteredSource, bbox, datetimeSelection, originalSource} = requestPayload;
-      let location = [];
-
-      if(categoryType === "Location"){
-          categoryValue = undefined;
-          location = this.state.selectedLocationCoordinates;
-      }
 
       SERVICES.FetchMessageSentences(siteKey, originalSource, bbox, datetimeSelection, timespanType, 
                                      limit, offset, edges, DEFAULT_LANGUAGE, Actions.DataSources(filteredSource), 
-                                     categoryValue?categoryValue.name.toLowerCase():categoryValue, searchValue, location, callback);
+                                     categoryValue?categoryValue.name.toLowerCase():categoryValue, searchValue, undefined, callback);
```

Comment:
I'm hesitant to pass in undefined as a parameter, since this might introduce new errors. Would there be a way to change the signature of this so it doesn't take a location parameter?